atom/atom.go

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
AFAIK, that doesn't matter - this is part of a creative work that was copyrighted in 2015, so that's when the clock started.

E.g., if Walt Disney adds a new brushstroke to Mickey Mouse, that doesn't change the copyright date for Mickey.